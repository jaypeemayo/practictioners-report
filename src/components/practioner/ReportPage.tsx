import * as React from "react";
import Report from "./Report";

export interface IReportPage {

}

export default class ReportPage extends React.Component<IReportPage, undefined> {
    render() {
        return (
            <>
                <h3>Report</h3>
                <Report />
            </>
        );
    }
}
