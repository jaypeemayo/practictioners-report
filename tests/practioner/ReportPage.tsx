import * as React from 'react';
import * renderer from 'react-test-renderer';
import ReportPage from '../../src/components/practioner/ReportPage';

it('should render ReportPage', () => {
    const component = renderer.create(
        <ReportPage/>,
      );
      let tree = component.toJSON();
      expect(tree).toMatchSnapshot();
});
