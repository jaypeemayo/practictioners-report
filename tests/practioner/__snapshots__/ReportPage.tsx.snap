// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ReportPage 1`] = `
Array [
  <h3>
    Practitioner Report
  </h3>,
  <div
    className="d-flex flex-column "
  >
    <div
      className="p-2"
    >
      <div
        className="d-flex flex-row flex-wrap"
      >
        <div
          className="p-2"
        >
          <strong>
             Search Practitioner:
          </strong>
        </div>
        <div
          className="p-2"
        >
          <input
            onChange={[Function]}
          />
        </div>
      </div>
      <div
        className="d-flex flex-row flex-wrap"
      >
        <div
          className="p-2"
        >
          <strong>
            Date Range:
          </strong>
        </div>
        <div
          className="p-2"
        >
          <div
            className="d-flex flex-row"
          >
            <div
              className=""
            >
              <div>
                <div
                  className="react-datepicker-wrapper"
                >
                  <div
                    className="react-datepicker__input-container"
                  >
                    <input
                      className=""
                      disabled={false}
                      onBlur={[Function]}
                      onChange={[Function]}
                      onClick={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      type="text"
                      value=""
                    />
                  </div>
                </div>
              </div>
            </div>
            <div
              className="pl-2"
            >
              <div>
                <div
                  className="react-datepicker-wrapper"
                >
                  <div
                    className="react-datepicker__input-container"
                  >
                    <input
                      className=""
                      disabled={false}
                      onBlur={[Function]}
                      onChange={[Function]}
                      onClick={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      type="text"
                      value=""
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="d-flex flex-row flex-wrap"
      >
        <div
          className="p-2"
        >
          <strong>
            Appointment Type:
          </strong>
        </div>
        <div
          className="p-2"
        >
          <input
            onChange={[Function]}
          />
        </div>
      </div>
    </div>
    <div
      className="p-2"
    />
  </div>,
]
`;
